src/app_builder.rs

```diff
@@ -1,69 +1,241 @@
-#[cfg(feature = "startup-stages")]
-use crate::startup_stage;
-use crate::{app::App, plugin::Plugin, stage};
+use crate::{TrackedValue, app::App, plugin::Plugin, tracked_unique::reset_tracked_unique, type_names::TypeNames};
 use shipyard::*;
 use std::{
     any::{type_name, TypeId},
+    borrow::Cow,
     collections::hash_map::Entry,
     collections::HashMap,
+    rc::Rc,
 };
 use tracing::*;
 
 mod plugin_id;
-mod workloads;
 use plugin_id::PluginId;
-use workloads::Workloads;
+
+/// Name of app stage responsible for doing most app logic. Systems should be registered here by default.
+pub const DEFAULT_STAGE: &str = "default";
```

Comment:
Why const and not static? (Reminder: const gets inlined everywhere, static gets put in program memory)

Reply:
Ah. I didn't know that. I'll change to static.